packages/sg721/src/lib.rs

```diff
@@ -92,31 +95,26 @@ pub struct UpdateCollectionInfoMsg<T> {
 }
 
 #[cw_serde]
-pub struct RoyaltyInfo {
-    pub payment_address: Addr,
+pub struct RoyaltyInfo<T: AddressLike> {
+    pub payment_address: T,
     pub share: Decimal,
+    pub updated_at: Timestamp,
```

Comment:
This will probably break de-serialization if a contract is migrating to this new version

Reply:
fixed w Option<Timestamp>